[drogon] Build error

Package: drogon[core,ctl,mysql,orm,redis]:x64-linux -> 1.8.4

**Host Environment**

- Host: x64-linux
- Compiler: GNU 13.1.1
-    vcpkg-tool version: 2023-06-15-8c254a5fb6c503615834fc66bd0717664a339a2e
    vcpkg-scripts version: 288e8bebf 2023-06-23 (4 days ago)

**To Reproduce**

`vcpkg install drogon[ctl,mysql,redis]`

**Failure logs**

```
Computing installation plan...
The following packages will be built and installed:
    drogon[core,ctl,mysql,orm,redis]:x64-linux -> 1.8.4
Installing 1/1 drogon:x64-linux...
Building drogon[core,ctl,mysql,orm,redis]:x64-linux...
-- Using cached an-tao-drogon-v1.8.4.tar.gz.
-- Extracting source /home/zhangqi/vcpkg/downloads/an-tao-drogon-v1.8.4.tar.gz
-- Applying patch vcpkg.patch
-- Applying patch drogon_config.patch
-- Using source at /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean
-- Found external ninja('1.11.1').
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/cmake --build . --config Debug --target install -- -v -j13
    Working Directory: /home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg
    See logs for more information:
      /home/zhangqi/vcpkg/buildtrees/drogon/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/drogon/portfile.cmake:39 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)
```
<details><summary>/home/zhangqi/vcpkg/buildtrees/drogon/install-x64-linux-dbg-out.log part 1</summary>

```
[1/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/CacheFile.cc.o -MF CMakeFiles/drogon.dir/lib/src/CacheFile.cc.o.d -o CMakeFiles/drogon.dir/lib/src/CacheFile.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/CacheFile.cc
[2/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/Cookie.cc.o -MF CMakeFiles/drogon.dir/lib/src/Cookie.cc.o.d -o CMakeFiles/drogon.dir/lib/src/Cookie.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/Cookie.cc
[3/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/DrTemplateBase.cc.o -MF CMakeFiles/drogon.dir/lib/src/DrTemplateBase.cc.o.d -o CMakeFiles/drogon.dir/lib/src/DrTemplateBase.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/DrTemplateBase.cc
[4/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/DrClassMap.cc.o -MF CMakeFiles/drogon.dir/lib/src/DrClassMap.cc.o.d -o CMakeFiles/drogon.dir/lib/src/DrClassMap.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/DrClassMap.cc
[5/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpFileUploadRequest.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpFileUploadRequest.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpFileUploadRequest.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpFileUploadRequest.cc
[6/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/AOPAdvice.cc.o -MF CMakeFiles/drogon.dir/lib/src/AOPAdvice.cc.o.d -o CMakeFiles/drogon.dir/lib/src/AOPAdvice.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/AOPAdvice.cc
[7/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpBinder.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpBinder.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpBinder.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpBinder.cc
[8/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/FiltersFunction.cc.o -MF CMakeFiles/drogon.dir/lib/src/FiltersFunction.cc.o.d -o CMakeFiles/drogon.dir/lib/src/FiltersFunction.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/FiltersFunction.cc
[9/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/ConfigLoader.cc.o -MF CMakeFiles/drogon.dir/lib/src/ConfigLoader.cc.o.d -o CMakeFiles/drogon.dir/lib/src/ConfigLoader.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/ConfigLoader.cc
[10/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpFileImpl.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpFileImpl.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpFileImpl.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpFileImpl.cc
[11/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpViewData.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpViewData.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpViewData.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpViewData.cc
[12/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpResponseParser.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpResponseParser.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpResponseParser.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpResponseParser.cc
[13/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpUtils.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpUtils.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpUtils.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpUtils.cc
[14/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpRequestParser.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpRequestParser.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpRequestParser.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpRequestParser.cc
[15/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpRequestImpl.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpRequestImpl.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpRequestImpl.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpRequestImpl.cc
[16/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpClientImpl.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpClientImpl.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpClientImpl.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpClientImpl.cc
[17/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/IntranetIpFilter.cc.o -MF CMakeFiles/drogon.dir/lib/src/IntranetIpFilter.cc.o.d -o CMakeFiles/drogon.dir/lib/src/IntranetIpFilter.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/IntranetIpFilter.cc
[18/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/RangeParser.cc.o -MF CMakeFiles/drogon.dir/lib/src/RangeParser.cc.o.d -o CMakeFiles/drogon.dir/lib/src/RangeParser.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/RangeParser.cc
[19/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/NotFound.cc.o -MF CMakeFiles/drogon.dir/lib/src/NotFound.cc.o.d -o CMakeFiles/drogon.dir/lib/src/NotFound.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/NotFound.cc
[20/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/LocalHostFilter.cc.o -MF CMakeFiles/drogon.dir/lib/src/LocalHostFilter.cc.o.d -o CMakeFiles/drogon.dir/lib/src/LocalHostFilter.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/LocalHostFilter.cc
[21/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpResponseImpl.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpResponseImpl.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpResponseImpl.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpResponseImpl.cc
[22/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/PluginsManager.cc.o -MF CMakeFiles/drogon.dir/lib/src/PluginsManager.cc.o.d -o CMakeFiles/drogon.dir/lib/src/PluginsManager.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/PluginsManager.cc
[23/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpSimpleControllersRouter.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpSimpleControllersRouter.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpSimpleControllersRouter.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpSimpleControllersRouter.cc
[24/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpServer.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpServer.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpServer.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpServer.cc
[25/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/ListenerManager.cc.o -MF CMakeFiles/drogon.dir/lib/src/ListenerManager.cc.o.d -o CMakeFiles/drogon.dir/lib/src/ListenerManager.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/ListenerManager.cc
[26/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/TaskTimeoutFlag.cc.o -MF CMakeFiles/drogon.dir/lib/src/TaskTimeoutFlag.cc.o.d -o CMakeFiles/drogon.dir/lib/src/TaskTimeoutFlag.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/TaskTimeoutFlag.cc
[27/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/HttpControllersRouter.cc.o -MF CMakeFiles/drogon.dir/lib/src/HttpControllersRouter.cc.o.d -o CMakeFiles/drogon.dir/lib/src/HttpControllersRouter.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/HttpControllersRouter.cc
[28/131] /usr/bin/c++ -DHAS_MYSQL_OPTIONSV -DHAS_YAML_CPP -DUSE_BROTLI -DUSE_OSSP_UUID=0 -DUSE_REDIS -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/orm_lib/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/nosql_lib/redis/inc -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg -I/home/zhangqi/vcpkg/buildtrees/drogon/x64-linux-dbg/exports -isystem /home/zhangqi/vcpkg/installed/x64-linux/include -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/uuid -isystem /home/zhangqi/vcpkg/installed/x64-linux/include/mysql -fPIC -g -std=c++17 -MD -MT CMakeFiles/drogon.dir/lib/src/RateLimiter.cc.o -MF CMakeFiles/drogon.dir/lib/src/RateLimiter.cc.o.d -o CMakeFiles/drogon.dir/lib/src/RateLimiter.cc.o -c /home/zhangqi/vcpkg/buildtrees/drogon/src/v1.8.4-be4cfc2af4.clean/lib/src/RateLimiter.cc
```
</details>

**Additional context**

--- a/ports/drogon/fix_gcc13.patch
+++ b/ports/drogon/fix_gcc13.patch
@@ -1,0 +1,12 @@
+diff --git a/lib/inc/drogon/RateLimiter.h b/lib/inc/drogon/RateLimiter.h
+index 6940321..2c5f039 100644
+--- a/lib/inc/drogon/RateLimiter.h
++++ b/lib/inc/drogon/RateLimiter.h
+@@ -3,6 +3,7 @@
+ #include <memory>
+ #include <chrono>
+ #include <mutex>
++#include <string>
+ 
+ namespace drogon
+ {

--- a/ports/drogon/portfile.cmake
+++ b/ports/drogon/portfile.cmake
@@ -7,6 +7,7 @@ vcpkg_from_github(
     PATCHES
         vcpkg.patch
         drogon_config.patch
+        fix_gcc13.patch #https://github.com/drogonframework/drogon/pull/1563
 )
 
 vcpkg_check_features(

--- a/ports/drogon/vcpkg.json
+++ b/ports/drogon/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "drogon",
   "version": "1.8.4",
+  "port-version": 1,
   "description": "A C++14/17 based HTTP web application framework running on Linux/macOS/Unix/Windows",
   "homepage": "https://github.com/an-tao/drogon",
   "documentation": "https://drogon.docsforge.com/master/overview/",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2182,7 +2182,7 @@
     },
     "drogon": {
       "baseline": "1.8.4",
-      "port-version": 0
+      "port-version": 1
     },
     "dstorage": {
       "baseline": "1.2.0",

--- a/versions/d-/drogon.json
+++ b/versions/d-/drogon.json
@@ -1,9 +1,14 @@
 {
   "versions": [
     {
+      "git-tree": "29376f140626e1d34d6ca573b5a00e6dd7006d2f",
+      "version": "1.8.4",
+      "port-version": 1
+    },
+    {
       "git-tree": "92af14d325c1fce661625e6adef2cf633acb5a37",
       "version": "1.8.4",
-      "port-version": 0 
+      "port-version": 0
     },
     {
       "git-tree": "670f02828b60f7a11c6b7a7feff0396d6cec4483",

